### PR TITLE
Candidate/v1.0

### PR DESCRIPTION
**Pull request by isocra**
- More fixes to JSHint
- Add `npm` version page
- Updated the README.md to have more useful instructions

### CODE REVIEW

**Comment by isocra** [comment]
@brendon Hi, I hope you had a good Christmas? If you have a mo, could you look through this PR? I've spent some time ironing out (I hope) the jshint config and the documentation and putting everything in place for installation via npm and yarn (which uses the same package repo). Thanks!!

File: README.md [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
+You'll also need to include [jQuery](https://jquery.com) before you include this plugin (so that jQuery is defined).
 
-`bower install https://github.com/isocra/TableDnD.git`
+---
 
-This will install the TableDnD plugin into your bower components as well as installing [jQuery](http://jquery.com) which it depends on.
+## Getting Started
 
-Documentation will go here, for now, go to [Isocra's TableDnD Blog posting](http://www.isocra.com/2008/02/table-drag-and-drop-jquery-plugin/)
+Let's create a simple table. The HTML for the table is very straight forward (no Javascript, pure HTML, we haven't added `thead` or `tbody` elements, but it works find with these too):
```
Comment (brendon) [inline README.md]: Should be "but it works **fine** with these" I think :)

File: dist/jquery.tablednd.js [hunk 2]
```diff
@@ -2,89 +2,15 @@
...
- * Version 0.6: 2011-12-02 Added support for touch devices
- * Version 0.7  2012-04-09 Now works with jQuery 1.7 and supports touch, tidied up tabs and spaces
+ * License: MIT.
+ * See https://github.com/isocra/TableDnD
  */
+
+/*jshint -W054 */
+
 !function ($, window, document, undefined) {
 // Determine if this is a touch device
```
Comment (brendon) [inline dist/jquery.tablednd.js]: You can drop this comment too since we don't determine this anymore :)

File: js/jquery.tablednd.js [hunk 3]
```diff
@@ -2,89 +2,15 @@
...
- * Version 0.6: 2011-12-02 Added support for touch devices
- * Version 0.7  2012-04-09 Now works with jQuery 1.7 and supports touch, tidied up tabs and spaces
+ * License: MIT.
+ * See https://github.com/isocra/TableDnD
  */
+
+/*jshint -W054 */
+
 !function ($, window, document, undefined) {
 // Determine if this is a touch device
```
Comment (brendon) [inline js/jquery.tablednd.js]: And this can be removed also.

**Comment by brendon** [comment]
I hope you had a great Christmas too! :D It's summer time here, so extended holidays for us.

**Comment by isocra** [comment]
Hi @brendon, updates done. I'm going to leave the code with as few changes as possible, so as not to disrupt people's code. I'm working on adding some tests and getting proper coverage and then I'll feel safer about changing stuff and it'll make it easier to accept PRs from other collaborators. But fitting this in might mean it happens in relatively slow time 😄!

**Comment by brendon** [comment]
I totally understand! :) I finally finished my contract work the day before Christmas so I'll have a bit more free time once I return to work in the new year. Ping me on your testing branch and I'll join in there.

**Pull request merged**